### ISSUE TITLE
Emit a BrightID "Sponsor" event

### ISSUE THREAD

**Issue by adamstallard** [clrfund/monorepo#136]
We can include in our clr.fund context settings a smart contract address that emits an event of this type:
```
event Sponsor(address);
```
The BrightID nodes will listen and sponsor anyone with an address included in such an event.

Any smart contract can do this; ideally the sponsor event should come after some minimal amount of engagement from a user--perhaps "signing up."

A user can't be verified unless they're sponsored, so it should precede this https://github.com/clrfund/monorepo/issues/135

**Comment by adamstallard** [comment]
```
 // sponsor any address provided as a parameter.
 function sponsor(address add) public {
   emit Sponsor(add);
 }
```

**Comment by xuhcc** [comment]
Implemented in #157
